index.js

```diff
@@ -2,21 +2,65 @@
 
 const express = require('express');
 const app = express();
+const { Client } = require('pg');
+const bodyParser = require('body-parser');
+app.use(
+  bodyParser.text({
+    type: 'text/html'
+  })
+);
 
-app.set('port', (process.env.PORT || 8080));
+const client = new Client({
+  connectionString: process.env.DATABASE_URL,
+  user: 'olamothe',
+  database: 'qa'
+});
+
+client.connect();
+
+app.set('port', process.env.PORT || 8080);
 app.use('/', express.static(__dirname + '/bin'));
 
+app.get('/qa/:pagename', function(request, response) {
+  client.query(`SELECT content FROM pages WHERE name LIKE $1 LIMIT 1;`, [request.params.pagename], (err, res) => {
+    if (err) {
+      response.send(err.message);
+    } else {
+      if (res.rowCount == 1) {
+        return response.send(res.rows[0].content);
+      } else {
+        return response.sendStatus(404);
+      }
+    }
+  });
+});
+
+app.post('/qa/:pagename', function(request, response) {
+  if (request.body) {
+    client.query(`SELECT 1 FROM pages WHERE name LIKE $1;`, [request.params.pagename], (err, res) => {
```

Comment:
It would be good to handle errors in this block. At the moment, if res.rowCount !== 1 because of an error or because no records exist, we do not distinguish and always send back a 200.